Makefile

```diff
@@ -12,7 +12,7 @@ BUILD_FLAGS = -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`g
 all: get_vendor_deps install test
 
 install:
-	CGO_ENABLED=0 go install $(BUILD_FLAGS) ./cmd/tendermint
+	go install $(BUILD_FLAGS) ./cmd/tendermint
 
 build:
 	CGO_ENABLED=0 go build $(BUILD_FLAGS) -o build/tendermint ./cmd/tendermint/
```

Comment:
maybe we should remove it from here too to be consistent?

Reply:
It doesn't have the same behavior on go build as per the comment. But sure, I can remove it. Will squish commits together